.github/workflows/azure-site-extension.yml

```diff
@@ -0,0 +1,79 @@
+name: Azure Site Extension
+
+on:
+  workflow_dispatch:
+
+env:
+  SPEC_FILE_TEMPLATE: 'NewRelic.Azure.WebSites.Extension.NodeAgent.nuspec'
+
+jobs:
+  setup_tools:
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup dotnet '6.0.x'
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: '6.0.x'
+      - name: Display dotnet version
+        run: dotnet --version
+      - name: Display NuGet version
+        run: nuget ?
+
+  create_extension_bundle:
+    runs-on: windows-latest
+    needs:
+      - setup_tools
+
+    strategy:
+      matrix:
+        node-version: ['lts/*']
```

Comment:
It's possible to deploy Node 18.x applications in Azure. Did we want to publish only for LTS?

Reply:
yes. our agent is node version agnostic. however, we may have to publish for both x86 and x64, [see](https://github.com/newrelic/node-newrelic/pull/2488#discussion_r1729341609)